lib/active_model/serializer/adapter.rb

```diff
@@ -1,12 +1,14 @@
 module ActiveModel
   class Serializer
     class Adapter
+      UnknownAdapterError = Class.new(ArgumentError)
```

Comment:
I only use the `Class.new` and `Struct.new` forms for dynamic class construction.

- `Foo = Struct.new` because otherwise you get superclass mismatch exceptions
- `Class.new` for one-liner Exception or sentinal objects, e.g. `BadObject = Class.new; if obj == BadOjbect....`.  (or when super serious meta-programming requires it).